getting-started-with-api/java-sdk/README.md

```diff
@@ -0,0 +1,51 @@
+# Getting Started Java SDK Examples
+Java applications illustrating topics from the [Getting Started](https://help.smartling.com/hc/en-us/articles/1260804661570-Getting-Started) tutorial.
+
+The following examples are included:
+
+* **Upload.java**. Upload a file to Smartling
+* **DownloadPseudo.java**. Download a pseudotranslated file from Smartling.
+
+## Prerequisites
+To run the sample code, you need:
+
+* Java installed
+* Maven installed
+* Smartling API token credentials for a test project in Smartling
+
+## Set Up
+The code expects the following environment variables to be defined:
+
+* **DEV_PROJECT_ID**. The UID of the Smartling test project.
+* **DEV_USER_IDENTIFIER**. The user identifier from your API token.
+* **DEV_USER_SECRET**. The user secret from your API token.
+
+To set these up, download a copy of the either `setenv.sh` (for Mac and Linux) or `setenv.cmd` (for Windows) from the root directory of the repository, update them with your own information, then run them in the terminal where you'll be running the examples:
+
+Mac or Linux
+```
+source ./setenv.sh
+```
+Windows
+```
+.\setenv.cmd
+```
+
+## Running the sample code
+The Maven configuration file, `pom.xml`, is configured with targets for running the examples.
+### Upload
```

Comment:
You probably should mention that strings should be authorized here or the first step in Download section. Otherwise, FAPI will not apply pseudo for not authorized strings. Or we have to keep `localeIdsToAuthorize` in code or add `authorize=true`

Reply:
It seems pseudo works also on unauthorized content.